src/app/page/UserProfile/UserOverview.jsx

```diff
@@ -51,7 +85,7 @@ const UserStatus = ({ profile, setProfile, view }) => {
       </Grid>
       <Grid item container spacing={1} direction="column">
         <Grid item container>
-          <H5>Address</H5>
+          <H5>Location</H5>
```

Comment:
If we're changing Address -> Location, we should probably update our variables to match to avoid confusion.

Reply:
I simply changed the label to reflect the design in Figma. We can keep Address instead of Location. I think it sounds more appropriate in a user profile.

Reply:
Not a deal breaker, for code review, but just for code clarity, I think this is something we should think about doing. This could create some confusion if we don't update language consistently across the app.